test/dummy/app/models/article.rb

```diff
@@ -1,2 +1,11 @@
 class Article < ActiveRecord::Base
+  belongs_to :admin_user
+
+  def computed
+    "Not orderable"
+  end
```

Comment:
Perhaps it should be named `non_orderable_column` then or something. So it's a bit more clear what the method is. Or if it's possible to add it only for the relevant test?